spec/bolt/transport/ssh_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'spec_helper'
 require 'net/ssh'
+require 'net/ssh/proxy/jump'
```

Comment:
I guess I missed adding this in when I looked for tests that were not standalone after refactoring loading code.